src/commcare_cloud/ansible/group_vars/all.yml

```diff
@@ -23,7 +23,7 @@ internal_network_interface_fact: "ansible_{{ internal_network_interface }}"
 internal_ipv4: "{{ hostvars[inventory_hostname][internal_network_interface_fact].ipv4 }}"
 
 shared_dir_gid: 1500  # This GID cannot already be allocated
-shared_dir_name: "shared{{ '_' ~ deploy_env if deploy_env != 'production' else '' }}"
+shared_dir_name: "shared1{{ '_' ~ deploy_env if deploy_env != 'production' else '' }}"
```

Comment:
perhaps you could change this in `staging/public.yml` (where it will only affect staging) instead of here (where it will affect all environments)?

Reply:
Updated variables - commcare-cloud/environments/staging/public.yml. At the time of migration in STG & PROD env, we will update the public.yml file.
Here I have mentioned **_efs_shared_mount_endpoint_** for EFS end-point address.